docs/Displaying a User Interface/Template Custom Buttons/index.md

```diff
@@ -77,8 +77,24 @@ sdlManager.getScreenManager().commit(new CompletionListener() {
 ```
 !@
 
+@![javascript]
+```js
+const textState = new SDL.manager.screen.utils.SoftButtonState('<#State Name#>', '<#Button Label Text#>', null);
+const softButtonObject = new SDL.manager.screen.utils.SoftButtonObject('softButtonObject', [textState], textState.getName(), function (softButtonObject, rpc) {
```

Comment:
This works for SoftButtonState but not SoftButtonObject since the `states` and `initialStateName` don't have setters and have to be passed to the constructor in order to be set.